examples/next/src/lib/app.ts

```diff
@@ -0,0 +1,8 @@
+/**
+ * Application config.
+ */
+const app = {
+  url: "https://storybook.animareflection.com",
```

Comment:
I meant to put this in the UI library config, but good call on keeping the examples atomic, I like this idea better and didn't think about that. Since this doesn't represent the "app URL" of the examples, I would change this (and the Vite example config) to be

```ts
const app = {
  uiLibraryUrl: "https://storybook.animareflection.com",
};
```

or `anirefUiLibraryUrl` or `storybookUrl` or whatever you think sounds best!